spyder_terminal/widgets/terminalgui.py

```diff
@@ -84,6 +86,11 @@ def setup_term(self):
         print("\0", end='')
         self.set_font(self.font)
         self.set_dir(self.initial_path)
+        options = self.parent.CONF.options('terminal')
```

Comment:
Please replace here `terminal` for `CONF_SECTION`, and import CONF_SECTION from `config.py`.